geode-core/src/distributedTest/java/org/apache/geode/internal/cache/PutAllGlobalDUnitTest.java

```diff
@@ -159,55 +159,56 @@ public void testputAllGlobalRemoteVM() throws Throwable {
 
     final int socketPort = vm0.invoke(PutAllGlobalDUnitTest::openSocket);
 
-    AsyncInvocation async1 = vm0.invokeAsync(PutAllGlobalDUnitTest::putAllMethod);
-
-    AsyncInvocation async2 = vm1.invokeAsync(new CacheSerializableRunnable("put from another vm") {
-      @Override
-      public void run2() throws CacheException {
-        long endTime = System.currentTimeMillis() + 5000;
-        boolean connected = false;
-        while (!connected && (System.currentTimeMillis() < endTime)) {
-          try {
-            Socket sock = new Socket(InetAddress.getLocalHost(), socketPort);
-            connected = true;
-            sock.close();
-          } catch (IOException ioe) {
-            // ignored - will time out using 'endTime'
+    AsyncInvocation<Void> async1 = vm0.invokeAsync(PutAllGlobalDUnitTest::putAllMethod);
+
+    AsyncInvocation<Void> async2 =
+        vm1.invokeAsync(new CacheSerializableRunnable("put from another vm") {
+          @Override
+          public void run2() throws CacheException {
```

Comment:
Use of a deprecated class can be avoided here by making this:
```
    AsyncInvocation<Void> async2 =
        vm1.invokeAsync("put from another vm", () -> {
          long endTime = System.currentTimeMillis() + 5000;
...
```
and removing the extra curly brace on line 210